core/src/main/codegen/templates/Parser.jj

```diff
@@ -6932,6 +6932,8 @@ SqlIdentifier ReservedFunctionName() :
     |   <NTILE>
     |   <NULLIF>
     |   <OCTET_LENGTH>
+    |   <PERCENTILE_CONT>
+    |   <PERCENTILE_DISC>
```

Comment:
nit: according to the sorting rule (e.g. `CHAR_LENGTH` comes before `CHARACTER_LENGTH`), the two new function names should come after `PERCENT_RANK`?

Reply:
Done.